mapie/control_risk/p_values.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Iterable, Union
+from scipy.stats import binom
+
+
+def hoefdding_bentkus_p_value(
+    r_hat: NDArray,
+    n: int,
+    alpha: Union[float, NDArray]
+) -> NDArray:
+    """
+    Parameters
+    ----------
+    r_hat : NDArray of shape (len(lambdas), )
+        Empirical risk of metric_control with respect
+        to the lambdas.
+    n : Integer value
+        Correspond to the number of observations in
+        X_cal dataset.
```

Comment:
ERROR: `X_cal` is not defined. Take another reference (or just say `the number of observations`).

Reply:
Done